src/main/kotlin/no/nav/syfo/consumer/dokarkiv/DokarkivConsumer.kt

```diff
@@ -44,7 +44,10 @@ class DokarkivConsumer(urlEnv: UrlEnv, private val azureAdTokenConsumer: AzureAd
                     log.info("Sending to dokarkiv successful, journalpost created")
                     response.body<DokarkivResponse>()
                 }
-
+                HttpStatusCode.Conflict -> {
```

Comment:
Er det successful når man får conflict?

Reply:
Ja, kan stort sett behandles som en 200 OK da det er den originale journalposten som blir returnert som body, men med httpstatus 409.

Reply:
Hm, ok. Hvorfor får vi det noen ganger? Er det dobbelsending fra oss eller noe?

Reply:
Ikke sikker at jeg er helt med på det du mente. 
Men poenget i denne PRen er at vi alltid setter den `eksternReferanseId` (vår intern id) i kallet mot dokarkiv for å unngå duplikate journalposter grunnet teknisk feil

Reply:
I hvert fall finner ingen errors med code 409 [ila siste 3 mnd](https://logs.adeo.no/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:60000),time:(from:now-90d%2Fd,to:now))&_a=(columns:!(level,message,envclass,application,pod),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:application,negate:!f,params:(query:esyfovarsel),type:phrase),query:(match_phrase:(application:esyfovarsel))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:level,negate:!f,params:(query:Error),type:phrase),query:(match_phrase:(level:Error)))),hideChart:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',interval:auto,query:(language:lucene,query:Dokarkiv),sort:!(!('@timestamp',desc)))).

Reply:
Ok! Lurte bare på hvordan feilen oppstår